### DISCUSSION TITLE
Cant Change board alignment

### DISCUSSION THREAD

**Discussion by Creepinsolo87** [iNavFlight/inav#6105] [Q&A]
Hello, i have Matek 405SE and i changed the board 90 degrees in configuration but when i go to setup to make sure it work, it still shows as default. i dont want to change my setup because it perfect and i have changed it before. So i dont know why im not able to now. Thank you for the help and ALL that you do!

[a comment by an automated account is not shown]

**Comment by FelixvdDonk** [comment]
Just stating the obvious but did you hit apply? The configuration tab requires you do this before you go into any other tab
